backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -418,4 +418,8 @@ SearchException conflictingIdentifierTypesForPredicate(ToDocumentIdentifierValue
 	@Message(id = ID_OFFSET_2 + 70,
 			value = "Text predicates (phrase, fuzzy, wildcard, simple query string) are not supported by this field's type.")
 	SearchException textPredicatesNotSupportedByFieldType(@Param EventContext context);
+
+	@Message(id = ID_OFFSET_2 + 71, value = "Projection on field `%1$s` is not allowed, since bridge did not specify a bind back for this field. " +
+			"Please define a projection converter or use .rawField(`%1$s`) to extract the raw values bypassing the bridge.")
```

Comment:
Same as for Elasticsearch, this would be more fitting:

```
"Projection on field '%1$s' is not allowed, since no projection converter was defined for this field. "+
"Please define a projection converter or use .rawField(\"%1$s\") to extract the raw values, bypassing the projection converter."
```